plugins/module_utils/occ.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, Marc Crébassa <aalaesar@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+import os
+# import json
```

Comment:
Better to not have commented code on main branch.

Reply:
cleaned up  the commented line